gradio/strings.py

```diff
@@ -19,6 +19,9 @@
     "COLAB_DEBUG_TRUE": "Colab notebook detected. This cell will run indefinitely so that you can see errors and logs. "
     "To turn off, set debug=False in launch().",
     "COLAB_DEBUG_FALSE": "Colab notebook detected. To show errors in colab notebook, set debug=True in launch()",
+    "COLAB_BETA": "\nNOTE: Using Embedded Colab Mode, a beta feature."
+    "\nIf you have issues, please revert to gradio==3.8.2 and file a Github issue. "
```

Comment:
If they encounter an issue, using a share link should work right? I think that'd be better than asking them to downgrade.

Reply:
Done